clusterloader2/cmd/clusterloader.go

```diff
@@ -47,12 +48,18 @@ func validateFlags() []error {
 
 func main() {
 	defer glog.Flush()
+	gomega.RegisterFailHandler(failHandler)
+	k8sframework.RegisterCommonFlags()
```

Comment:
Instead of registering all these flags yourself, could you just import `kubernetes/test/e2e` package which does it automatically as part of init?

Reply:
Or just call `framework.ViperizeFlags()`